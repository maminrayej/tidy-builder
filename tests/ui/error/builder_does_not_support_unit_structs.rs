#[derive(tidy_builder::Builder)]
struct MyStruct;

fn main() {}
