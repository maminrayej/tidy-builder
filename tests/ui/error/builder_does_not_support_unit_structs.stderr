error: Builder does not support unit structs
 --> tests/ui/error/builder_does_not_support_unit_structs.rs:1:10
  |
1 | #[derive(tidy_builder::Builder)]
  |          ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `tidy_builder::Builder` (in Nightly builds, run with -Z macro-backtrace for more info)
